test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlWriterTests.cs

```diff
@@ -2460,6 +2460,84 @@ private void WriteAndVerifyXml(IEdmModel model, string expected, CsdlTarget targ
             }
         }
 
+        [Fact]
+        public void VerifyKeyAliasForComplexPropertiesIsSupported()
+        {
+            // Arrange
+            var model = new EdmModel();
+            var entity = new EdmEntityType("NS1", "Product");
+            var complexType = new EdmComplexType("NS1", "ExampleComplexObject");
+            entity.AddStructuralProperty("ExampleObject", new EdmComplexTypeReference(complexType, isNullable: false));
+            entity.AddStructuralProperty("Name", EdmCoreModel.Instance.GetString(false));
+            entity.AddStructuralProperty("UpdatedTime", EdmCoreModel.Instance.GetDate(false));
+            var keyProperty = complexType.AddStructuralPropertyAlias("Name", EdmCoreModel.Instance.GetString(false), "ExampleObjectName", new List<string> { "ExampleObject", "Name" });

```

Comment:
I think it's redundant to have both `Name` and `{ ExampleObject, Name }` as parameters since you can retrieve the property name from the latter.